Void uses the "None" build type to apply our chosen compile settings.

diff --git a/runtime/lua/vim/health/health.lua b/runtime/lua/vim/health/health.lua
index dd6fe7f..25d8d81 100644
--- a/runtime/lua/vim/health/health.lua
+++ b/runtime/lua/vim/health/health.lua
@@ -157,7 +157,7 @@ local function check_performance()
   local buildtype = vim.fn.matchstr(vim.fn.execute('version'), [[\v\cbuild type:?\s*[^\n\r\t ]+]])
   if buildtype == '' then
     health.error('failed to get build type from :version')
-  elseif vim.regex([[\v(MinSizeRel|Release|RelWithDebInfo)]]):match_str(buildtype) then
+  elseif vim.regex([[\v(MinSizeRel|Release|RelWithDebInfo|None)]]):match_str(buildtype) then
     health.ok(buildtype)
   else
     health.info(buildtype)
